src/20231103_AaveV3Ethereum_TreasuryManagementAddToRETHHolding/TreasuryManagementAddToRETHHolding_20231103.s.sol

```diff
@@ -0,0 +1,63 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
+
+import {GovV3Helpers, IPayloadsControllerCore, PayloadsControllerUtils} from 'aave-helpers/GovV3Helpers.sol';
+import {EthereumScript} from 'aave-helpers/ScriptUtils.sol';
+import {AaveV3Ethereum_TreasuryManagementAddToRETHHolding_20231103} from './AaveV3Ethereum_TreasuryManagementAddToRETHHolding_20231103.sol';
+import {WethEthMockChainlinkOracle} from './WethEthMockChainlinkOracle.sol';
+/**
+ * @dev Deploy Ethereum
+ * command: make deploy-ledger contract=src/20231103_AaveV3Ethereum_TreasuryManagementAddToRETHHolding/TreasuryManagementAddToRETHHolding_20231103.s.sol:DeployEthereum chain=mainnet
+ */
+contract DeployEthereum is EthereumScript {
+  function run() external broadcast {
+    // deploy payloads
+    AaveV3Ethereum_TreasuryManagementAddToRETHHolding_20231103 payload0 = new AaveV3Ethereum_TreasuryManagementAddToRETHHolding_20231103();
+
+    // compose action
+    IPayloadsControllerCore.ExecutionAction[]
+      memory actions = new IPayloadsControllerCore.ExecutionAction[](1);
+    actions[0] = GovV3Helpers.buildAction(address(payload0));
+
+    // register action at payloadsController
+    GovV3Helpers.createPayload(actions);
+  }
+}
+
+/**
+ * @dev Deploy Mock
+ * command: make deploy-ledger contract=src/20231103_AaveV3Ethereum_TreasuryManagementAddToRETHHolding/TreasuryManagementAddToRETHHolding_20231103.s.sol:DeployMock chain=mainnet
+ */
+contract DeployMock is EthereumScript {
```

Comment:
This script should provably be removed. If its needed, then, some comments on the purpose would be good